src/main/java/org/prebid/server/bidder/yieldlab/YieldlabBidder.java

```diff
@@ -110,6 +95,25 @@ private ExtImpYieldlab constructExtImp(List<ExtImpYieldlab> extImps) {
         return ExtImpYieldlab.builder().adslotId(adSlotIdsParams).targeting(targeting).build();
     }
 
+    private List<ExtImpYieldlab> collectImpExt(List<Imp> imps) {
+        final List<ExtImpYieldlab> extImps = new ArrayList<>();
+        for (Imp imp : imps) {
+            final ExtImpYieldlab extImpYieldlab = parseImpExt(imp);
+            if (Objects.nonNull(extImpYieldlab)) {
```

Comment:
It is not friendly to throw npe here which will fall the whole auction.
May be let's cover by condition?

Pls review `Objects.nonNull(..)` usages.

Reply:
Not the issue, confused by `requireNonNull` method.